package.json

```diff
@@ -12,7 +12,7 @@
     "start": "yarn stripes serve",
     "build": "yarn stripes build --output ./output",
     "test": "yarn run test:unit",
-    "test:unit": "jest --ci --coverage",
+    "test:unit": "jest --ci --coverage --updateSnapshot",
```

Comment:
Sure. This was a workaround to fix the failing snapshots. Because the github action couldn't resolve the snapshot issue even though the tests passed locally.

Reply:
update tests locally and commit them

Reply:
It doesn't help. I have done it twice already

Reply:
But ` "test:unit": "jest --ci --coverage --updateSnapshot",` did the track

Reply:
stripes-components could be updated after your commits

Reply:
I am on it. I have just merged